src/Orchard.Web/Modules/Orchard.MediaLibrary.WebSearch/Module.txt

```diff
@@ -0,0 +1,23 @@
+﻿Name: Orchard.MediaLibrary.WebSearch
+AntiForgery: enabled
+Author: The Orchard Team
+Website: https://orchardproject.net
+Version: 1.0
+OrchardVersion: 1.10.3
+Description: Enables Orchard to use image search in media or as a provider.
+Features:
+    Orchard.MediaLibrary.WebSearch:
+        Description: Enables Orchard to use image search in media or as a provider.
+        Category: Media
+        Name: Media Library Web Search
+        Dependencies: Orchard.MediaLibrary
+    Orchard.MediaLibrary.WebSearch.Bing:
+        Description: Enables Orchard to use Bing image search in media or as a provider.
```

Comment:
Bing web search provider for importing images to the Media Library.
(similarly for Pixabay too)